src/main.rs

```diff
@@ -1047,6 +1061,57 @@ mod tests {
         });
     }
 
+    #[test]
+    fn test_vunet() {
+        test_block!(tb, "", {
+            let mut clear = ClearDiskConfig::new();
+            let guest = Guest::new(&mut clear);
+
+            let mut workload_path = dirs::home_dir().unwrap();
+            workload_path.push("workloads");
+
+            let mut vubridge_path = workload_path.clone();
+            vubridge_path.push("vubridge");
+            let vubridge_path = String::from(vubridge_path.to_str().unwrap());
+
+            // Start the daemon
+            let mut daemon_child = Command::new(vubridge_path.as_str()).spawn().unwrap();
+
+            let mut cloud_child =
+                Command::new("/root/cloud-hypervisor/target/debug/cloud-hypervisor")
```

Comment:
This is why you're getting permission denied. on the CI.

After fixing this I see an error like this:
```
rob@rbradford-test:~/cloud-hypervisor$ cargo test --features "integration_tests" -- test_vunet
    Finished dev [unoptimized + debuginfo] target(s) in 0.05s
     Running target/debug/deps/cloud_hypervisor-f60d64920772e0c1

running 1 test
ud socket: /tmp/vubr.sock (server)
local:     127.0.0.1:4444
remote:    127.0.0.1:5555
Waiting for connections on UNIX socket /tmp/vubr.sock ...
Added sock 3 for watching. max_sock: 3
bind: Address already in use
Cloud Hypervisor Guest
	vCPUs: 4
	Memory: 512 MB
	Kernel: "/home/rob/workloads/hypervisor-fw"
	Kernel cmdline: 
	Disk(s): Some([DiskConfig { path: "/tmp/ch.3Sb9u4S7Nuan/osdisk.img" }, DiskConfig { path: "/tmp/ch.3Sb9u4S7Nuan/cloudinit" }])
Guest boot failed: Can not create a new virtual machine: DeviceManager(CreateVhostUserNet(VhostUserCreateMaster(VhostUserProtocol(SocketConnect(Os { code: 111, kind: ConnectionRefused, message: "Connection refused" })))))
test tests::test_vunet ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 20 filtered out

```

But your test still passes?